components/base/Collapsible.tsx

```diff
@@ -76,6 +79,7 @@ function Collapsible({ triggerText, children }: Props) {
         trigger={closedTrigger}
         triggerWhenOpen={openTrigger}
         transitionTime={transitionTime}
+        tabIndex={0}
```

Comment:
Adds the `tabIndex` attribute to the trigger element which in turn allows the Collapsible trigger to gain focus.